core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -2734,6 +2737,15 @@ protected void doAddInternalRole(String roleName, String[] userList,
                         finalValues.put(claim, value);
                     }
                 }
+            } else {
+                if (property == null && claim.equals(DISAPLAY_NAME_CLAIM)) {
```

Comment:
use as following for avoiding possible null pointer exceptions.

DISPLAY_NAME_CLAIM.equals(claim)